### PR TITLE
fix: exclude datasets with multiple organisms

### PR DESCRIPTION
**Pull request by MDunitz**
### Reviewers
**Functional:** 

**Readability:** 

---

## Changes

- modify dataset selection function to exclude datasets with multiple organisms

### CODE REVIEW

File: backend/wmg/data/extract.py [hunk 1]
```diff
@@ -29,20 +28,23 @@ def get_dataset_s3_uris():
...
-            session.query(Dataset.table.id, Dataset.table.assay)
+            session.query(Dataset.table.id, Dataset.table.assay, Dataset.table.organism)
             .join(Dataset.table.collection)
             .filter(
                 Dataset.table.assay != "null",
                 Dataset.table.published == "TRUE",
                 Dataset.table.is_primary_data == "PRIMARY",
                 Collection.table.visibility == "PUBLIC",
                 Dataset.table.tombstone == "FALSE",
+                Dataset.table.organism != "null",
```
Comment (atolopko-czi) [inline backend/wmg/data/extract.py:39]: do the null and empty organisms cases also need tests?

**Review by atolopko-czi — APPROVED** [review]
LGTM, though possibly missing tests for edge cases (see comment)

File: backend/wmg/data/extract.py (hunk 1 above)
Comment (MDunitz) [reply backend/wmg/data/extract.py:39]: our schema should check for that so its unlikely to make it to the db but I noticed that null cases could break the db query for assays so I figured I would include the check for organisms too.
added a test case just to be safe

**Review by ebezzi — APPROVED** [review]
